app/models/vote.rb

```diff
@@ -82,5 +82,11 @@ def evaluate_reward(new_vote, value, subject)
       end
       reward
     end
+
+    def voted(resource, user)
+      vote = where(user_id: user.id, voteable_id: resource.id, voteable_type: resource.class.to_s).first
```

Comment:
But you know the user's vote record is only created when the user makes a vote. I think the query will return `nil` when no record of that match is found. Or does `select` work in a different way? 
cc @andela-ooranagwa, @andela-oakinniranye